net-dns/dnsdist/files/dnsdist.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=dnsdist
+Wants=network-online.target
+After=network-online.target
```

Comment:
Are you sure you require the system to be online? I think most of the DNS servers are entirely happy with the network interfaces being up.